com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=dri"
+
+sdk-extensions:
+  - 'org.freedesktop.Sdk.Compat.i386'
+  - 'org.freedesktop.Sdk.Extension.toolchain-i386'
+
+add-extensions:
+    'org.freedesktop.Platform.Compat.i386':
+        directory: "lib32"
```

Comment:
The compat extension generally is intended to be mounted at `lib/i386-linux-gnu`.